server/src/infra/graphql/mutation.rs

```diff
@@ -120,6 +122,44 @@ impl Success {
     }
 }
 
+struct UnpackedAttributes {
+    email: Option<Email>,
+    display_name: Option<String>,
+    attributes: Vec<DomainAttributeValue>,
+}
+
+fn unpack_attributes(
+    attributes: Vec<AttributeValue>,
+    schema: &PublicSchema,
+    is_admin: bool,
+) -> FieldResult<UnpackedAttributes> {
+    let email = attributes
+        .iter()
+        .find(|attr| attr.name == "mail")
+        .cloned()
+        .map(|attr| deserialize_attribute(&schema.get_schema().user_attributes, attr, is_admin))
+        .transpose()?
+        .map(|attr| attr.value.unwrap::<String>())
+        .map(Email::from);
+    let display_name = attributes
+        .iter()
+        .find(|attr| attr.name == "displayName")
```

Comment:
Mm... excuse the question, but isn't the attribute called "display_name"? https://github.com/lldap/lldap/blob/6cf0f6df0639425f792af243b4345acc9b737fa8/server/src/domain/schema.rs#L75-L76

Reply:
Yup, it is... Sending a fix.